apps/drec-ui/src/apps/device-group/view/pages/UnreservedPage/UnreservedPage.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC } from 'react';
+import { CircularProgress, Paper, Typography, Grid, Button, Box } from '@material-ui/core';
+import { useStyles } from './UnreservedPage.styles';
+import { useUnreservedPageEffects } from './UnreservedPage.effects';
+import { GenericForm, TableComponent } from '@energyweb/origin-ui-core';
+
+export const UnreservedPage: FC = () => {
+    const classes = useStyles();
+    const {
+        deviceGroups,
+        tableProps,
+        formData,
+        mobileView,
+        isLoading,
+        noUnreservedDeviceGroupsTitle,
+        onReserveHandler,
+        disableReserveButton
+    } = useUnreservedPageEffects();
+
+    if (isLoading) {
+        return <CircularProgress />;
+    }
+
+    return (
+        <Paper className={classes.paper}>
+            <Typography variant="h5">Unreserved Device Groups</Typography>
+            <GenericForm
+                inputsVariant="filled"
+                twoColumns={!mobileView}
```

Comment:
If I am not mistaken Generic Form has a built in behaviour for switching from DoubleColumn to SingleColumn on smaller screen sizes. So here only `true` can be supplied